css/index.css

```diff
@@ -348,24 +361,79 @@ label {
 /* View style for tablets/ipads */
 @media screen and (max-width: 999px) {
   .iframe {
-    right: 0; /* Adjust the positioning of your iframe for tablet */
+    right: 0;
     left: 0;
   }
 
+  .direction-buttons {
+    position: absolute;
+    top: 20px;
+    right: 5px;
+    list-style: none;
+    padding-left: 90px;
+    font-size: small;
+  }
+  .direction-buttons li {
+    margin-right: 5px; /* Adjust the margin to reduce space between buttons */
+  }
+
   .welcome {
-    padding-top: 150px; /* Adjust the padding for the welcome section on tablet */
+    padding-top: 150px;
     padding-bottom: 250px;
-    font-size: xx-large; /* Adjust the font size for tablet */
+    font-size: xx-large;
+    margin-left: 50px;
+    margin-right: 20px;
+  }
+
+  .about-body {
+    margin-left: 50px;
+    margin-right: 20px;
+    padding-top: 200px;
   }
 
-  .about-body,
   .xp {
-    margin-left: 20px; /* Adjust the margins for about-body and xp sections on tablet */
+    padding-top: 200px;
+    margin-left: 50px;
+    margin-right: 20px;
+  }
+  .xp-item {
+    padding: 2%;
+    border: none;
+  }
+
+  #skills ul {
+    margin-left: 50px;
+    margin-right: 20px;
+  }
+
+  #projects ul {
+    justify-content: center;
+    margin-left: 50px;
     margin-right: 20px;
   }
 
   .connect-images {
-    left: 10%; /* Adjust the positioning of the connect images on tablet */
+    margin-left: 60px;
+    margin-right: 10px;
+  }
+
+  .leave_message fieldset {
+    margin-left: 50px;
+    margin-right: 20px;
+  }
+  #messages {
+    margin-left: 50px;
+    margin-right: 20px;
+  }
```

Comment:
you might want to give messages a min-height or some bottom margin to give it some space between itself and the footer
<img width="452" alt="Screen Shot 2023-09-21 at 11 00 28 PM" src="https://github.com/CristianMoran1/freyr-intro/assets/12771651/6933a6d2-2efb-4efd-9ce2-5722d57b7c09">